src/helpViewer/index.ts

```diff
@@ -620,6 +703,28 @@ function pimpMyHelp(helpFile: HelpFile): HelpFile {
         });
     }
 
+    // Highlight help preview:
+    if(helpFile.isPreview){
+        let rdInfo: string;
+        if(helpFile.rdPath){
+            const localRdPath = vscode.workspace.asRelativePath(helpFile.rdPath);
+            const rdUri = vscode.Uri.file(helpFile.rdPath);
+            const cmdUri = makeWebviewCommandUriString('vscode.open', rdUri);
+            rdInfo = `<a href="${cmdUri}" title="Open File">${localRdPath}</a>`;
+        } else{
+            rdInfo = `an .Rd file`;
+        }
+        if(helpFile.rPath){
+            const localRPath = vscode.workspace.asRelativePath(helpFile.rPath);
+            const rUri = vscode.Uri.file(helpFile.rPath);
+            const cmdUri = makeWebviewCommandUriString('vscode.open', rUri);
```

Comment:
The command uri here does not work for me.

When I click the hyperlink to the file, vscode cannot find the file:

![image](https://user-images.githubusercontent.com/4662568/202624166-9f526d43-6bda-4bef-9b40-c4e858307a55.png)

Reply:
I added a check, that makes sure the linked file actually exists. Do you still get failing links with the latest commit?

Reply:
I tried with the same package (RMariaDB), and it works on macOS and Linux. But on Windows, it never works for me. The file does exists, but it seems vscode does not accept the uri as a valid one.

<img width="506" alt="image" src="https://user-images.githubusercontent.com/4662568/202851706-650d2c4e-5b50-40bc-b89b-824b01f1016f.png">

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/4662568/202851740-a39667d8-b548-4332-b7f6-465ae1c9b0c8.png">

Reply:
The file path vscode is trying to open in the lower pane seems to be an absolute, linux-style path (`home/ken/Workspaces/RMariaDB/R/default.R`), which surprises me, since it should be a relative path, as in the top pane (`./R/default.R`).

Does the top of the corresponding Rd file say `% Please edit documentation in R/default.R` or is there a different path in there?

Do the path variables [here](https://github.com/REditorSupport/vscode-R/blob/06b13fb23c52cdacc4139b521b7a6823f18e7667/src/helpViewer/index.ts#L719-L722) point to the expected places (`helpFile.rPath`: absolute path of R file, `localRPath`: same file, but local w.r.t. workspace root, `rUri`: vscode.Uri of the same file)?

(I checked with a fresh clone of RMariaDB on windows and all links are working for me)

Reply:
Forgot to tell, It is in Remote WSL.

Reply:
I tried in Windows and everything works nicely. The problem only occurs in Remote WSL.

Reply:
The issue was apparently caused by the fact that the vscode-R extension runs on the remote machine (WSL) and webviews run on the UI machine (windows) and therefore the URI embedded in the help page was interpreted as a windows one. I added a custom command instead of the default open command, which should fix this.

Reply:
It works nicely on WSL now 👍